transform/transformer.go

```diff
@@ -72,7 +79,23 @@ func trimFirstRune(s string) string {
 func (t *LogTransformer) transformDiff() {
 	lines := strings.Split(t.LogContent, "\n")
 	var output []string
+	var numberOfDifferencesString string
+	var numberOfReplaces int
 	for _, line := range lines {
+		// https://regex101.com/r/XtxJgT/1
+		regexNumberOfDifferencesString := regexp.MustCompile(`Number of stacks with differences:.*`)
+		matchesNumberOfDifferencesString := regexNumberOfDifferencesString.FindStringSubmatch(line)
+		if matchesNumberOfDifferencesString != nil {
+			numberOfDifferencesString = matchesNumberOfDifferencesString[0]
+		}
+
+		// https://regex101.com/r/DGpxCX/1
+		regexNumberOfReplaces := regexp.MustCompile(`(requires replacement)`)
```

Comment:
Besides require replacement there is also "may cause replacement".  We could also add it to count of resources. Here is one example of a diff 
```
Resources
[~] AWS::RDS::DBParameterGroup SpmArchive/ParameterGroup SpmArchiveParameterGroupA3C62C14 replace
 ├─ [~] Description (requires replacement)
 │   ├─ [-] Parameter group for mysql5.7
 │   └─ [+] Parameter group for mysql8.0
 ├─ [~] Family (requires replacement)
 │   ├─ [-] mysql5.7
 │   └─ [+] mysql8.0
 └─ [~] Parameters
     ├─ [-] Removed: .query_cache_size
     ├─ [-] Removed: .query_cache_type
     └─ [~] .sql_mode:
         ├─ [-] only_full_group_by,strict_trans_tables,error_for_division_by_zero,no_auto_create_user,no_engine_substitution
         └─ [+] only_full_group_by,strict_trans_tables,error_for_division_by_zero,no_engine_substitution
[~] AWS::RDS::DBInstance SpmArchive/Primary SpmArchivePrimary8D438481 may be replaced
 ├─ [~] DBParameterGroupName (may cause replacement)
 │   └─ [~] .Ref:
 │       ├─ [-] SpmArchiveParameterGroupA3C62C14
 │       └─ [+] SpmArchiveParameterGroupA3C62C14 (replaced)
 └─ [~] EngineVersion
     ├─ [-] 5.7.38
     └─ [+] 8.0.34


✨  Number of stacks with differences: 1
```
Additionally I guess you intended to include the round brackets when searching for string. It needs to be escaped otherwise it counts as group match.
Here is my proposal 
```suggestion
		regexNumberOfReplaces := regexp.MustCompile(`\(requires replacement\)|\(may cause replacement\)`)
```

Reply:
updated